contracts/clients/src/BlsWalletWrapper.ts

```diff
@@ -120,6 +127,15 @@ export default class BlsWalletWrapper {
     return this.ExpectedAddress(verificationGateway, pubKeyHash);
   }
 
+  static async getRandomBlsPrivateKey(): Promise<string> {
+    await mcl.init(mcl.BN_SNARK1);
```

Comment:
Hey @jacque006, I was curious to get your thoughts on this.  I want the `getRandomBlsPrivateKey` function to be static but I need to make sure `.init` has been called on mcl.  Looks like the only way we can guarantee that right now is if I put this code in the signer (I originally did that and then pulled it out because I didn't think it makes sense).  Do you think it's an issue if I import the mcl library in this file?

Reply:
I would try to keep any logic like this in the signer dir, and I would use `@thehubbleproject/bls` over a direct `mcl-wasm` import.

You should be able to do something like this:

`./contracts/clients/signers/getRandomPrivateKey.ts`
```typescript```
import { init, randFr } from "@thehubbleproject/bls";

export default function getRandomPrivateKey(): Promise<string> {
    await init();
    return `0x${randFr().serializeToHexStr()}`;
}
```

Then add as an export to `./contracts/clients/signers/index.ts` and consume here.

Reply:
You might run into issues running `mcl.init` twice. If that happens will have to try another way.

Reply:
This worked for me.  Thanks, @jacque006 !